TransmartCoreGrailsPlugin.groovy

```diff
@@ -95,13 +95,13 @@ A runtime dependency for tranSMART that implements the Core API
             conceptsResourceService(DefaultConceptsResource)
         }
 
-        context.'component-scan'('base-package': 'org.transmartproject.db.dataquery.highdim') {
+        context.'component-scan'('base-package': 'org.transmartproject.db') {
```

Comment:
I don't think this is correct. You won't find subclasses of `AbstractHighDimensionDataTypeModule` outside of ` 'org.transmartproject.db.dataquery.highdim'`,